app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_instances = [
+        Person(person["name"], person["age"]) for person in people
+    ]
+
+    for ind in people:
+        """ind -> individual ;P; in other case the line"s too long"""
```

Comment:
You can create a variable for `Person.people[ind["name"]]` to use descriptive names